### PR TITLE
DDF-939 - Completed implementation of optional banners in search ui

### PR DESCRIPTION
**Pull request by McLucas**
-Hooked in new Platform UI configurations into the search ui.
-added extra ajax call for the ui configurations.
-noticed drawhelper.js was using twipsy and was overlayed over my footer.  Displaying it by default fixes it and it will appears when you are in draw mode (good news).
-added ui config to the test config (fixed tests i broke during this
ticket)
-changes no-value option to "(no value)" as requested.

### CODE REVIEW

**Comment by McLucas** [comment]
@millerw8 you might be interested in this.

**Comment by shaundmorris** [comment]
:+1:  would like to pull in today if there are no objections.

**Comment by shaundmorris** [comment]
LOL that's a Geico commercial now...

File: search-ui/standard/src/test/resources/ui.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "footer": "UNCLASSIFIED",
```
Comment (kcwire) [inline search-ui/standard/src/test/resources/ui.json:2]: These kind of defaults don't belong in DDF. Contact me directly if you want to get into the specifics. but this should be backedout and removed.

**Comment by shaundmorris** [comment]
I agree with part of that and will discuss offline.

**Pull request merged**